src/common/loading-indicator/loading-indicator.ts

```diff
@@ -1,4 +1,4 @@
-import { LoadingIndicatorStyles } from './embedded-checkout-styles';
+import { LoadingIndicatorStyles } from '../../embedded-checkout';
```

Comment:
Can you move the `LoadingIndicatorStyles` to the `common/loading-indicator` folder as well?

Reply:
moved